app/src/main/java/edu/mills/cs115/MainActivity.java

```diff
@@ -23,6 +23,17 @@
 import com.google.android.gms.appindexing.Thing;
 import com.google.android.gms.common.api.GoogleApiClient;
 
+/**
+ * The top-level activity for the CS63 dictionary switch. It allows users to launch
+ * {@link HomeFragment}, {@link HistoryListFragment}, and {@link AZCategoryFragment}.
+ * It Contains the search view and the bottom navigation bar. It also displays a
+ * meme carousel fragment and a "Word of the Day" fragment.
+ *
+ * @author barango
+ * @author Roberto Amparán (mr.amparan@gmail.com)
+ * @author Ying Parks
+ * @author Ashley Vo
+ */
```

Comment:
+ The top-level activity that manages the navigation between fragments and implements the view for the entire application?? something like that. Mentioning the carousel and W.O.T.D. is redundant, as it will be in Ying's javadoc.